config/initializers/clowder_config.rb

```diff
@@ -44,15 +49,20 @@ def self.instance
         options["databasePort"] = ENV['DATABASE_PORT']
         options["databaseUsername"] = ENV['DATABASE_USER']
         options["databasePassword"] = ENV['DATABASE_PASSWORD']
+        options["endpoints"] = {
+          "ingress-services" => "https://ci.cloud.redhat.com",
```

Comment:
This will have to be moved to an env variable later, right?

Reply:
I just make it the same as before the change. I guess yes we could it an ENV.

Reply:
changed to ENV